packages/@okta/vuepress-site/docs/guides/build-sso-integration/specify-your-settings/openidconnect/settings.md

```diff
@@ -24,13 +26,13 @@
   3. Change the **Login initiated by** field to **Either Okta or App** to give your integration an Okta tile.
       >**Note:** When you select this option, an **App Embed Link** section appears at the bottom of the page with the URL that can be used to sign in to the OIDC client from outside of Okta.
   4. Check the box for **Display application icon to users**.
```

Comment:
Step 4: If this is a check box (I think so), use "Select" instead of "Check" as per Microsoft Manual of Style.

Reply:
Change made